sunpy/net/vso/tests/test_vso.py

```diff
@@ -479,24 +451,11 @@ def test_build_client_params():
         build_client(url="http://notathing.com/")
 
 
-@pytest.mark.remote_data
```

Comment:
Why has this test gone?

Reply:
This test and the test below were introduced since 2.1 they are testing a fix for functionality which has been broken since 1.0. Since I just removed the need for `VSOClient().search(...).search(...)` to work any more I deleted the method, fix and tests.